### PR TITLE
English updates

### PR DESCRIPTION
**Pull request by xmrhaelan**
Minor edits, additions, and new turpial description file based on the Spanish version that's currently linked to google docs. Probably best to move as much outside of GitHub on to GitHub so new contributors can support current development and have an easier time catching themselves up.

### CODE REVIEW

**Comment by rdymac** [comment]
Thank you for the corrections and for adding the Turpial description from the Google Doc. 

Please, next time make two different Pull Requests to separate the review process of each part.

File: documents/turpial-description.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+This would be the minimum system, 4MB of storage and battery management. A disadvantage is the low RAM they carry. They lack of PSRAM could have an impact on the number of users who simultaneously use the access point "AP".
+ 
+It can be purchased on the manufacturer's own website, www.adafruit.com or at any local or global distributor such as mouser:
+ https://www.mouser.es/ProductDetail/Adafruit/3405?qs=sGAEpiMZZMuJ3l9lTgMBp1ZUJQFPkBG2kgGNDfvEbt9%252BtBjtG%2Fb8ag%3D%3D
+
+### Heltec WiFi 32 V1 and Heltec WiFi 32 V2
+The first has a storage of 4MB, the second of 8MB (more than enough). It includes a charging and battery management system, it also has a small OLED screen of 128x64 points that could be used to monitor some parameters without having to connect it to a PC.
+
+V1
+https://www.amazon.es/MakerHawk-desarrollo-Bluetooth-pantalla-pulgadas/dp/B076P8GRWV/ref=sr_1_1?__mk_es_ES=%C3%85M%C3%85%C5%BD%C3%95%C3%91&crid = 1I6U573FU3RRH & keywords = heltec% 2Blora & qid = 1567694750 & s = gateway & sprefix = helte% 2Caps% 2C154 & sr = 8-1 & th = 1
```
Comment (jeandudey) [inline documents/turpial-description.md]: Link broken.

File: documents/turpial-description.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+V2
+https://www.amazon.es/MakerHawk-desarrollo-Bluetooth-pantalla-pulgadas/dp/B076T28KWG/ref=sr_1_1?__mk_es_ES=%C3%85M%C3%85%C5%BD%C3%95%C3%91&crid = 1I6U573FU3RRH & keywords = heltec% 2Blora & qid = 1567694750 & s = gateway & sprefix = helte% 2Caps% 2C154 & sr = 8-1 & th = 1
```
Comment (jeandudey) [inline documents/turpial-description.md]: Link broken.

**Comment by jeandudey** [comment]
@luisan00 are Heltec boards still used?

**Pull request merged**